### PR TITLE
[imp]replace the constant JDK_DEFAULT_REJECT_HANDLER init static code

### PR DESCRIPTION
**Pull request by fkinjava**
[imp]replace the constant JDK_DEFAULT_REJECT_HANDLER init static code in BaseDeployments

### CODE REVIEW

**Comment by fkinjava** [comment]
[imp]replace the constant JDK_DEFAULT_REJECT_HANDLER init static code in BaseDeployments

File: restlight-server/src/main/java/esa/restlight/server/BaseDeployments.java [hunk 1]
```diff
@@ -73,14 +73,13 @@
...
-                TimeUnit.SECONDS,
-                new LinkedBlockingQueue<>(),
-                new RestlightThreadFactory("useless"));
-        JDK_DEFAULT_REJECT_HANDLER = e.getRejectedExecutionHandler().getClass();
-        e.shutdownNow();
+        Class<? extends RejectedExecutionHandler> defaultHandlerClass = ThreadPoolExecutor.AbortPolicy.class;
+        try {
+            defaultHandlerClass = (Class<? extends RejectedExecutionHandler>) ThreadPoolExecutor.class.getDeclaredField("defaultHandler").getType();
+        } catch (NoSuchFieldException e) {
+            LoggerUtils.logger().error("Could not find the field named defaultHandler in ThreadPoolExecutor, JDK_DEFAULT_REJECT_HANDLER will be set to ThreadPoolExecutor$AbortPolicy.class", e);
```
Comment (alalag1) [inline restlight-server/src/main/java/esa/restlight/server/BaseDeployments.java:80]: What about printing the log by debug-level and fallbacking to `ThreadPoolExecutor.getRejectedExecutionHandler().getClass()` if failed to retrieve `JDK_DEFAULT_REJECT_HANDLER` by reflection

**Review by alalag1 — CHANGES_REQUESTED** [review]

**Comment by alalag1** [comment]
@fkinjava there are checkstyle errors, take a look plz

**Comment by alalag1** [comment]
@fkinjava thx a lot

**Pull request merged**